pkg/controller/apiclient/apiclient_util_test.go

```diff
@@ -14,18 +14,180 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apiclient
+package apiclient_test
```

Comment:
@krancour Ah that's a good point; I'll move this to the mem package.

Can someone please direct me to the reaction chain fakes? I can't seem to find anything about it..